tools/ratbag-command.c

```diff
@@ -101,6 +101,8 @@ usage(void)
 	       "Profile Commands:\n"
 	       "  profile active get		Print the currently active profile\n"
 	       "  profile active set N		Set profile N as to the  active profile\n"
+	       "  profile enabled get           Check if profile is enabled\n"
+	       "  profile enabled set E         Set profile enabled state to E\n"
```

Comment:
needs to be updated in the manpage too, and I'm really undecided about this here. I'm not a fan of parse_bool down below, we should only accept one option instead of trying to guess what the user meant.

I think it's worth here to add a new parsing command so we can say "profile enable" and "profile disable". It's a bit more work but I think long-term it pays off and it makes the command more natural (and would also work for resolution enable/disable, etc.)